### PR TITLE
fix drop-menu tests, add initial bookmarks-widget tests

### PR DESCRIPTION
**Pull request by jbwyme**
Not done adding tests for the bookmarks widget but this is a functional place to pause and get a review + merge

### CODE REVIEW

File: lib/components/drop-menu/index.js [hunk 1]
```diff
@@ -57,17 +57,17 @@ registerMPElement('mp-drop-menu', class extends Component {
...
       } else {
-        this.closeMenu();
+        this._closeMenu();
       }
     }
   }
 
   attachedCallback() {
     super.attachedCallback(...arguments);
     if (this.isAttributeEnabled('open')) { // open the menu if it's not explicitly closed
```
Comment (tdumitrescu) [inline lib/components/drop-menu/index.js]: is this comment outdated?

File: test/widgets/bookmarks-widget.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/* global beforeEach, describe, it */
+import expect from 'expect.js';
+import 'webcomponents.js/webcomponents';
```
Comment (tdumitrescu) [inline test/widgets/bookmarks-widget.js]: How come the component tests no longer need this polyfill but this widget test does?

File: test/widgets/bookmark-data.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "user_id": 3,
+    "include_in_dashboard": true,
+    "user": "Ted D.",
+    "name": "Ted's most excellent adventure"
+}, {
+    "id": 10,
+    "user_id": 3,
+    "include_in_dashboard": false,
+    "user": "Ted D.",
+    "name": "Down and to the left baby"
```
Comment (tdumitrescu) [inline test/widgets/bookmark-data.json]: you know it

File: test/widgets/bookmarks-widget.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+import '../../lib/widgets/bookmarks-widget';
+import bookmarks from './bookmark-data.json';
+
+const waitForAnimation = cb => {
+  setTimeout(cb, 500);
```
Comment (tdumitrescu) [inline test/widgets/bookmarks-widget.js]: lulz. any better way?

File: test/widgets/bookmarks-widget.js (hunk 2 above)
Comment (jbwyme) [inline test/widgets/bookmarks-widget.js]: it doesn't, missed removing it

File: test/widgets/bookmarks-widget.js (hunk 4 above)
Comment (jbwyme) [inline test/widgets/bookmarks-widget.js]: well.... maybe... I'll try something

File: lib/components/drop-menu/index.js (hunk 1 above)
Comment (jbwyme) [inline lib/components/drop-menu/index.js]: yes

**Pull request merged**